frontend/javascripts/oxalis/geometries/materials/plane_material_factory.ts

```diff
@@ -697,17 +763,18 @@ class PlaneMaterialFactory {
     // layers were least-recently activated (but are now disabled).
     // The first array contains the color layer names and the second the segmentation layer names.
```

Comment:
What does the third return value represent? From the code it looks like it is the overall number of layers, but then, shouldn't that also be returned in line 766?

Reply:
I think the case where `0` is returned doesn't really matter as nothing will be rendered, anyway, but it's probably less confusing to return the global layer count. I'll adapt the code (and add a comment) :)